code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -192,7 +192,7 @@
 	new /obj/item/holosign_creator/security(src)
 	new /obj/item/reagent_containers/spray/pepper(src)
 	new /obj/item/clothing/suit/armor/vest/det_suit(src)
-	new /obj/item/storage/belt/holster/full(src)
+	//new /obj/item/storage/belt/holster/full(src) disabled due to change in holsters

```

Comment:
should be `sunset - disabled due to change in holsters`

Reply:
No problem I completely agree and will do comments like this going forward.